RFS/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,31 @@
+status = WARN
+
+property.logsDir = ${env:TUPLE_DIR_PATH:-./logs/tuples}
+
+appenders = console, FailedRequests
+
+appender.console.type = Console
+appender.console.name = Console
+appender.console.target = SYSTEM_OUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC} %p %c{1.} [%t] %m%n
+
+appender.FailedRequests.type = RollingRandomAccessFile
+appender.FailedRequests.name = FailedRequests
+appender.FailedRequests.fileName = ${logsDir}/failedRequests.log
```

Comment:
This looks like it's going to have issues with multiple RFS workers writing to the same path and files right?

Reply:
Ugh - handling this scenario is going to require more work, its a little convoluted to extract the container identifier.  I'll create a follow up PR that handles the deployment changes.